src/main/java/io/strimzi/kafka/bridge/MetricsReporter.java

```diff
@@ -24,6 +25,7 @@
      * @param jmxCollectorRegistry JmxCollectorRegistry instance for scraping metrics from JMX endpoints
      * @param meterRegistry MeterRegistry instance for scraping metrics exposed through Vert.x
      */
+    @SuppressFBWarnings("SIC_INNER_SHOULD_BE_STATIC_ANON")
```

Comment:
why this one?

Reply:
Same here, checkstyle / spotbugs complained ...

Reply:
ditto as above

Reply:
Same ... as you can see, I didn't change anything else in this class.